install/kots/manifests/kots-config.yaml

```diff
@@ -6,7 +6,7 @@ spec:
   groups:
     - name: general
       title: General settings
-      description: General settings for your Gitpod instance
+      description: General settings for your Gitpod instance. Please make sure to have a [running cert-manager instance](https://cert-manager.io/docs/installation/) before proceeding.
```

Comment:
I would have a simple boolean, defaulting to `false` (and, if possible, requiring `true`) that says "cert-manager is installed on this system"